README.md

```diff
@@ -192,9 +222,31 @@ s3:
+The options provided in the command line take precedence over those provided in any config file that gets passed in. With config file now supporting both S3 and IAM Role include/exclude filters, it is important to bear in mind the following scenarios to avoid confusion or unexpectedly delete resources.
+
+```shell
+cloud-nuke aws --config path/to/config.yaml
+```
+
+Running this command will result in the nuking of _ALL_ resources, using config file for filtering only S3 and/or IAM Role resources specified. Any unsupported resources in the config file, such as `ec2` will be ignored, and if filters for IAM role and/or S3 are not present, cloud-nuke will delete all S3 and IAM role resources.
+
+```shell
+cloud-nuke aws --resource-type s3 --config path/to/config.yaml
+```
+
+Here `cloud-nuke` used along with `--resource-type s3` option will result in nuking S3 buckets. Given a `--config` option, any include/exclude filters for S3 will be used to perform the deletion. Filters for IAM Role will be ignored(?). Any unsupported resources will be ignored, and if S3 resource filters are not present in config file, cloud-nuke will delete all S3 buckets. 
+
+```shell
+cloud-nuke aws --resource-type iam-role --config path/to/config.yaml
+```
+
+Similar to previous example, `cloud-nuke` used along with `--resource-type iam-role` option will result in nuking IAM roles. Given a `--config` option, any include/exclude filters for IAM roles will be used to perform the deletion. Filters for S3 will be ignored(?). Any unsupported resources will be ignored, and if IAM role resource filters are not present in config file, cloud-nuke will delete all IAM roles.
+
+```shell
+cloud-nuke aws --resource-type s3 --resource-type iam-role --config path/to/config.yaml
+```
 
-In the same vein, say you do not provide a `--resource-type` option in the command line, but you do pass in a config file that only lists rules for `s3:`, such as `cloud-nuke aws --config path/to/config.yaml`. In this case _all_ resources would be nuked, but among `s3` buckets, only those matching your config file rules would be nuked.
+In this case, `cloud-nuke` used along with `--resource-type s3` and `--resource-type iam-role` options will result in nuking S3 buckets _PLUS_ IAM roles. Given a `--config` option, any include/exclude filters for S3 and/or IAM roles will be used to perform the deletion. Any unsupported resources will be ignored, and if IAM role and/or S3 resource filters are not present in config file, cloud-nuke will delete all IAM roles and S3 buckets.
```

Comment:
I think at this point it is clear what happens with the `--resource-type` filter, so I think you can simplify this section to:

```
You can also combine `--resource-type` filters to apply to multiple resources. For example, if you wish to delete both S3 buckets and IAM roles based on the config file:

\`\`\`shell
 cloud-nuke aws --resource-type s3 --resource-type iam-role --config path/to/config.yaml 
\`\`\`

```

Reply:
Agree with this suggestion

Reply:
Updated with suggested changes.